src/pages/ReportDetailsPage.js

```diff
@@ -76,6 +77,17 @@ function ReportDetailsPage(props) {
 
     const isGroupDMChat = useMemo(() => ReportUtils.isDM(props.report) && participants.length > 1, [props.report, participants.length]);
 
+    const isPrivateNotesFetchTriggered = !_.isUndefined(props.report.isLoadingPrivateNotes);
+
+    useEffect(() => {
```

Comment:
We have a hook `WithReportAndPrivateNotesOrNotFound` to fetch the private note, I think we can use that, right?

Reply:
But we need to add more props to ```WithReportAndPrivateNotesOrNotFound``` to not display ```FullScreenLoadingIndicator``` or ```NotFoundPage``` because this hook have the logic:
https://github.com/Expensify/App/blob/63014263a542633621da9efd7617b5f9d0879fcc/src/pages/home/report/withReportAndPrivateNotesOrNotFound.js#L89-L104
what do you think?

Reply:
That might change behavior. The logic used in our current method is quite different and might not be suitable after replacing HOC. IMO we don't expect to see a 'not found' screen.

Reply:
To clarify, changing the logic in the HOC won't reflect what it does based on its name.

Reply:
> That might change behavior

So I think we should keep the current logic rather than using ```WithReportAndPrivateNotesOrNotFound```

Reply:
I think that's fine, then. Thanks for having a look.